golem_messages/message/tasks.py

```diff
@@ -483,6 +483,19 @@ class SubtaskResultsAccepted(TaskMessage):
         'report_computed_task': base.MessageSlot(ReportComputedTask),
     }
 
+    @classmethod
+    def deserialize_with_header(cls, header, data, *args, **kwargs):  # noqa pylint: disable=arguments-differ
+        instance: SubtaskResultsAccepted = super().deserialize_with_header(
+            header, data, *args, **kwargs
+        )
+        instance.is_valid()
+        return instance
+
+    def is_valid(self) -> bool:
```

Comment:
and, maybe indeed the base signature should be changed so that it's recognized `is_valid` can raise a `ValidationError`

Reply:
@shadeofblue, afaik, there's no support for declaring throwing of exceptions in python. Or did you mean sth else?

Reply:
yeah, I mean the docstring syntax and actually, I see that it's already there ;)

```
        :raises: `exceptions.ValidationError`
```